package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "@kubernetes/client-node",
-      "version": "0.16.1",
+      "version": "0.16.3",
```

Comment:
@brendanburns Do we always bump versions for patches? I am a little out of the loop with dependabot here.

Reply:
I think that dependabot needs to rebase this.